### ISSUE TITLE
PR to ESPHome Code Base

### ISSUE THREAD

**Issue by bzumik1** [lanwin/esphome_samsung_ac#9]
Hi, is there plan to merge this component to ESPHome code base? I think it would be nice touch. I fully understand that we need to test it properly first but I think that with this functionality we have right now it would be nice 1.0 version.

**Comment by lanwin** [comment]
I am absolutely not sure about that. I did not know what requirements they have to do that. Maybe later. But for now I would like to focus on adding some features and fixing issues.

**Comment by bzumik1** [comment]
I totally agree, I just wanted to open this so at point when we will feel comfortable it si working we would create PR to ESPHome. I have seen some other AC brands are already there. But this would be just nice thing to have, I agree it has lowest priority.

**Comment by TCB13** [comment]
+1 on this. Thank you.
